wowup-electron/src/assets/i18n/de.json

```diff
@@ -241,57 +241,57 @@
     "OPTIONS": {
       "APPLICATION": {
         "CURRENT_LANGUAGE_LABEL": "Aktuelle Sprache",
-        "ENABLE_SYSTEM_NOTIFICATIONS_DESCRIPTION": "Aktivieren / Deaktivieren verschiedener Systembenachrichtigungen",
+        "ENABLE_SYSTEM_NOTIFICATIONS_DESCRIPTION": "Aktivierung/Deaktivierung verschiedener Systembenachrichtigungen",
         "ENABLE_SYSTEM_NOTIFICATIONS_LABEL": "Systembenachrichtigungen aktivieren",
-        "MINIMIZE_ON_CLOSE_DESCRIPTION_MAC": "Beim schließen WowUp in der Menübar minimieren",
+        "MINIMIZE_ON_CLOSE_DESCRIPTION_MAC": "Beim Schließen des WowUp-Fensters in die Menübar minimieren.",
         "MINIMIZE_ON_CLOSE_DESCRIPTION_WINDOWS": "Beim Schließen des WowUp-Fensters auf das Systemabschnitt minimieren.",
-        "MINIMIZE_ON_CLOSE_LABEL": "Minimieren beim Schliessen",
-        "SET_LANGUAGE_CONFIRMATION_DESCRIPTION": "Zum Ändern der Standardsprache muss die Anwendung neu gestartet werden.",
-        "SET_LANGUAGE_CONFIRMATION_LABEL": "Festlegen einer neuen Standardsprache",
-        "SET_LANGUAGE_DESCRIPTION": "Wähle eine Sprache aus, in die du wechseln möchtest",
+        "MINIMIZE_ON_CLOSE_LABEL": "Beim Schließen minimieren",
+        "SET_LANGUAGE_CONFIRMATION_DESCRIPTION": "Die Änderung der Standardsprache erfordert einen Neustart.",
+        "SET_LANGUAGE_CONFIRMATION_LABEL": "Neue Standardsprache wird eingestellt",
+        "SET_LANGUAGE_DESCRIPTION": "Wähl eine Sprache aus, in die Du wechseln möchtest",
         "SET_LANGUAGE_LABEL": "Sprache einstellen",
-        "START_MINIMIZED_DESCRIPTION": "...und wird nicht auf dem Bildschirm angezeigt",
-        "START_MINIMIZED_LABEL": "Starte WowUp minimiert",
-        "START_WITH_SYSTEM_DESCRIPTION": "WowUp wird direkt gestartet, nachdem dein Betriebssystem geladen wurde...",
-        "START_WITH_SYSTEM_LABEL": "Starte WowUp mit dem System",
-        "TELEMETRY_DESCRIPTION": "Helfen Sie WowUp zu verbessern, indem Sie anonyme Installationsdaten und/oder Fehler senden.",
+        "START_MINIMIZED_DESCRIPTION": "WowUp wird minimiert und nicht angezeigt gestartet.",
+        "START_MINIMIZED_LABEL": "WowUp minimiert starten",
+        "START_WITH_SYSTEM_DESCRIPTION": "WowUp wird beim Starten des Systems gestartet.",
+        "START_WITH_SYSTEM_LABEL": "WowUp mit dem System starten",
+        "TELEMETRY_DESCRIPTION": "WowUp helfen, indem anonyme Installationsdaten und/oder Fehler gesendet werden.",
         "TELEMETRY_LABEL": "Telemetrie",
-        "THEME_DESCRIPTION": "Ändere das Farbschma nach deinen Wünschen",
+        "THEME_DESCRIPTION": "Die Farbschema nach Herzenslust verändern",
```

Comment:
"Ändere das Farbschma nach deinen Wünschen"
defenetly not "Herzenslust"

Reply:
I agree that nach deinen Wünschen would actually be better, but I don't think that the description of the feature should include "ändere" and "das Farbschma". "Ändere" is as if you're commanding the user to do so (it's just a description of what you are able to do), and "das Farbschma" is just wrong since it's "die Farbschema".

Reply:
Eben auf deutsch: 
Die App hat mehrere Farbschemen ja, aber nur eins kann aktiv sein, sprich man ändert das aktuelle Farbschema und das ist Singular. Man ändert das aktuelle Farbschema der App.

Reply:
Ich verstehe nun, danke!